### DISCUSSION TITLE
re：viewを使用してPDFファイルをPDFに変換します。

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1833] [Q&A]
PDFファイルを持っていてre：view内に置き、それをPDFファイルにエクスポートすることは可能ですか（出力はre：viewからのPDFです）。
つまり、re：viewを使用してPDFファイルをPDFに変換します。

**Comment by kdmsnr** [comment]
PDF同士を結合したほうが良さそうです。

**Comment by kmuto** [comment]
kdmsnrさんのいうように、目的によってはPDFにしてから結合したほうが簡単かもしれません。

rake pdf(review-pdfmaker)の範囲でなんとかする方法(挿絵などを入れる用途)としては、

1. imagesフォルダに貼り付けたいPDFファイルを置いておく。たとえばimages/insert.pdfという3ページぶんのPDFがあるとする。
2. reファイルでは、以下のようにTeX埋め込みで全ページ貼り付けを行う。

```
〜〜

//embed[latex]{
\includefullpagegraphics*[page=1]{images/insert.pdf}
\includefullpagegraphics*[page=2]{images/insert.pdf}
\includefullpagegraphics*[page=3]{images/insert.pdf}
//}

〜〜
```

`*`があるときにはページ番号(ノンブル)が入ります。`*`なしにするとノンブルは入りません。
